src/plugins/home/server/services/sample_data/data_sets/otel/index.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { i18n } from '@osd/i18n';
+import path from 'path';
+import { AppLinkSchema, SampleDatasetSchema } from '../../lib/sample_dataset_registry_types';
+import {
+  appendDataSourceId,
+  getSavedObjectsWithDataSource,
+  overwriteSavedObjectsWithWorkspaceId,
+} from '../util';
+import { logsFieldMappings } from './logs_field_mappings';
+import { metricsFieldMappings } from './metrics_field_mappings';
+import { servicesFieldMappings } from './services_field_mappings';
+import { tracesFieldMappings } from './traces_field_mappings';
+
+const otelDataName = i18n.translate('home.sampleData.otelSpecTitle', {
+  defaultMessage: 'Sample Open-Telemetry logs, Traces and Metrics',
+});
+const otelDataDescription = i18n.translate('home.sampleData.otelSpecDescription', {
+  defaultMessage:
+    'Sample data including correlated observability signals for an e-commerce application in Open-Telemetry standard.',
+});
+const initialAppLinks = [] as AppLinkSchema[];
```

Comment:
nit. avoid assertions if possible
```suggestion
const initialAppLinks: AppLinkSchema[] = [];
```

Reply:
Updated to add traces and services links [c6949c4](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8587/commits/c6949c4d965ee3542887cbe64baeb84361a8616d)